packages/pcd-types/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pcd/pcd-types",
-  "version": "0.6.0",
+  "version": "0.7.0",
```

Comment:
don't update this manually - i have a script (`./new-version.sh`) that invokes a library that bumps these versions whenever I publish the packages to npm